package.json

```diff
@@ -167,6 +169,24 @@
           "ui-organizations.creds.view"
         ]
       },
+      {
+        "permissionName": "ui-organizations.acq.unit.assignment.assign",
```

Comment:
isn't it too many dots? IMHO, `ui-organizations.acqUnits.assign` looks better

Reply:
copy/paste from orders/invoices). But I agree `ui-organizations.acqUnits.assign` looks better